AFTests/AlgoStore/AlgoStoreTestsInstanceRequired.cs

```diff
@@ -1011,24 +1011,24 @@ public partial class AlgoStoreTestsInstanceRequired : CreateAlgoWithInstanceFixt
             }
 
             // Wait up to 3 minutes for the algo to be started
-            await AlgoStoreCommonSteps.WaitAlgoToStart(ClientInstanceRepository, postInstanceData);
+            await WaitAlgoInstanceToStart(postInstanceData.InstanceId);
 
             // Check StoppingEntity row exists for Live and Demo and does not exist for Test instance
-            stoppingEntity = await AlgoStoreCommonSteps.GetStoppingEntityForInstance(ClientInstanceRepository, postInstanceData);
+            stoppingEntity = await GetStoppingEntityForInstance(ClientInstanceRepository, postInstanceData);
             Assert.That(stoppingEntity != null, Is.EqualTo(expectedResult));
 
             // Stop the algo instance
-            await AlgoStoreCommonSteps.StopAlgoInstance(Consumer, postInstanceData);
+            await StopAlgoInstance(postInstanceData);
 
             // Check StoppingEntity row does not exists for all instance types
-            stoppingEntity = await AlgoStoreCommonSteps.GetStoppingEntityForInstance(ClientInstanceRepository, postInstanceData);
+            stoppingEntity = await GetStoppingEntityForInstance(ClientInstanceRepository, postInstanceData);
             Assert.That(stoppingEntity != null, Is.False);
```

Comment:
you can simplify that using  Assert.That(stoppingEntity != null, Is.False); =>  Assert.That(stoppingEntity, Is.Null);

Reply:
I'll update it in the next PR. It must be copy/paste from the asserts before that.